tests/test_auth_email_login.py

```
@@ -0,0 +1,107 @@
+
+    def test_login_by_email(self, tmp_path):
+        mgr = AuthManager(tmp_path)
+        mgr.setup_user("alice", "Alice", "alice@example.com", "pass1234!")
+        ok, record = mgr.check_password("pass1234!", username="alice@example.com")
+        assert ok is True
+        assert record["username"] == "alice"
+
+    def test_login_by_email_case_insensitive(self, tmp_path):
+        mgr = AuthManager(tmp_path)
+        mgr.setup_user("alice", "Alice", "alice@example.com", "pass1234!")
+        ok, record = mgr.check_password("pass1234!", username="ALICE@EXAMPLE.COM")
```

Ruff (0.15.15)

[warning] Unpacked variable `record` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)